crates/wit-parser/src/lib.rs

```diff
@@ -266,14 +285,24 @@ pub struct IncludeName {
 
 /// The key to the import/export maps of a world. Either a kebab-name or a
 /// unique interface.
-#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize)]
+#[serde(into = "String")]
 pub enum WorldKey {
     /// A kebab-name.
     Name(String),
     /// An interface which is assigned no kebab-name.
     Interface(InterfaceId),
```

Comment:
Shoot. Forgot to make this one a JSON number like the other enums.

Reply:
Looks like this is what we want, because `WorldKey` is used in `World` exports and imports `IndexMap<WorldKey, WorldItem>`.